src/shared-components/banner/index.tsx

```diff
@@ -34,7 +34,11 @@ type BannerProps = {
  *
  * Banners are not dismissable.
  */
-export const Banner = ({ content, type = 'default', onClick }: BannerProps) => {
+export const Banner = ({
+  content,
+  type = 'default',
+  onClick = undefined,
```

Comment:
[Nit] Some not-yet-converted-to-TypeScript files needed to mark optional functions as undefined due to TypeScript inference of JavaScript files--it looks like we don't need it here, though. Can we remove the explicit undefined?

Reply:
done